File: leoz-node/src/test/java/org/deku/leoz/node/auth/IdentityTest.java
package org.deku.leoz.node.auth;

import org.deku.leoz.Identity;
import org.deku.leoz.SystemInformation;
import org.deku.leoz.bundle.BundleType;
import org.junit.Test;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

/**
 * Created by masc on 27.06.15.
 */
public class IdentityTest {
    Logger mLog = LoggerFactory.getLogger(this.getClass());

    @Test
    public void testIdentity() {
        Identity ident = Identity.Companion.create(BundleType.LEOZ_NODE.getValue(), SystemInformation.create());
        mLog.info(ident.toString());
    }
}

File: leoz-common/src/test/java/org/deku/leoz/IdentityTest.kt
package org.deku.leoz

import com.esotericsoftware.yamlbeans.YamlWriter
import org.deku.leoz.Identity
import org.deku.leoz.SystemInformation
import org.deku.leoz.bundle.BundleType
import org.junit.Test
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import org.yaml.snakeyaml.Yaml
import org.yaml.snakeyaml.constructor.Constructor
import org.yaml.snakeyaml.introspector.PropertyUtils
import org.yaml.snakeyaml.representer.Representer
import java.io.StringWriter

/**
 * Created by masc on 27.06.15.
 */
class IdentityTest {
    private var log = LoggerFactory.getLogger(this.javaClass)

    val identity by lazy {
        Identity.create(BundleType.LEOZ_NODE.value, SystemInformation.create())
    }

    @Test
    fun testIdentity() {
        log.info(this.identity.toString())
    }
}
